.travis.yml

```
@@ -7,14 +7,14 @@ os:
   - osx
```

a question for the maintainers - but osx seem overkill here

---

great feedback! committed changes. lets see what the maintainers say about 
```suggestion
  - osx
```